spec/rubocop/options_spec.rb

```diff
@@ -239,6 +244,24 @@ def abs(path)
       end
     end
 
+    describe '--no-parallel' do
+      around(:each) do |example|
+        create_file('.rubocop', '--parallel')
+        example.run
+      ensure
+        FileUtils.rm('.rubocop')
+      end
```

Comment:
I don't think we need to create a `.rubocop` file here, we test that it is superceded by command line options here: https://github.com/rubocop/rubocop/blob/6f6efd08aa806ee00eede93df60b38479b00841c/spec/rubocop/options_spec.rb#L492-L506

I think we can just test that `--no-parallel` sets parallel to false and that'd be enough.